docs/public/theme.css

```diff
@@ -53,31 +52,46 @@
   --theme-code-inline-text: var(--theme-text);
   --theme-code-bg: var(--color-gray-700);
   --theme-code-text: var(--color-gray-100);
+  --theme-navbar-bg: var(--color-orange);
+  --theme-navbar-height: 3.5rem;
+  --theme-sidebar-offset: var(--theme-navbar-height);
 }
 
 body {
   background: var(--theme-bg);
   color: var(--theme-text);
 }
 
+:root.scrolled {
+  --theme-sidebar-offset: 0;
+}
+
 :root.theme-dark {
   color-scheme: dark;
-  --theme-accent-opacity: 0.3;
+  --theme-accent-opacity: 0.4;
+  --theme-accent: var(--color-orange);
+  --theme-text-accent: var(--color-orange);
+  --theme-accent-rgb: var(--color-orange-rgb);
   --theme-divider: var(--color-gray-900);
   --theme-text: var(--color-gray-200);
   --theme-text-light: var(--color-gray-300);
   --theme-text-lighter: var(--color-gray-600);
-  --theme-text-accent: var(--color-white);
   --theme-bg: var(--color-gray-800);
   --theme-bg-hover: var(--color-gray-600);
   --theme-bg-offset: var(--color-gray-950);
-  --theme-code-inline-bg: var(--color-gray-600);
+  --theme-code-inline-bg: var(--color-gray-900);
   --theme-code-inline-text: var(--color-white);
   --theme-code-bg: var(--color-gray-950);
   --theme-code-text: var(--color-white);
+  --theme-navbar-bg: var(--color-purple);
 }
 
 ::selection {
-  color: var(--theme-text-accent);
-  background-color: rgba(var(--theme-accent-rgb), var(--theme-accent-opacity));
```

Comment:
Could you define new theme colors like a `--theme-selection` variable that's defined up above so that all of the theme colors are defined in one place?